src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -24,6 +24,7 @@ def get(self, request: Request, project) -> Response:
         """
         Fetches alert rules and legacy rules for a project
         """
+        expand = request.GET.getlist("expand", [])  # type: ignore
```

Comment:
I ignored the typing here because I needed to put `List[str]` but that meant I had to import `List` but then linting yelled because it wasn't used.